header.tpl

```diff
@@ -0,0 +1,13 @@
+ * Copyright {{ year-regexp }} cloud-provider-kubevirt Authors.
```

Comment:
Can you please elaborate about this file, why is it needed?

Reply:
There is another linter (not activated here) that check the golang source code files has the proper header, this is important for upstream projects.

Reply:
Removed header file.